packages/@magic-sdk/provider/src/core/sdk.ts

```diff
@@ -118,6 +118,7 @@ export interface MagicSDKAdditionalConfiguration<
   testMode?: boolean;
   deferPreload?: boolean;
   useStorageCache?: boolean;
+  meta?: any; // Generic field for clients to add metadata
```

Comment:
Could you provide an example of well-structured metadata that our client, WC, has previously submitted? I'm interested in understanding the standards and quality they adhere to.

Reply:
They'll be submitting something like this:

meta: {
  projectId: "abcd123"
}

Reply:
But the idea is that the meta field is unopinionated, so that any client can put identifying info in there to reference in event logs later on. They could for example put in their internal user ID, or classification tags.